### ISSUE TITLE
Travis builds are broken on dev branch

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/namecoin-core#142]
Starting December 26, the `dev` branch has been failing on Travis CI.  See the Travis build history: https://travis-ci.org/namecoin/namecoin-core/builds

**Comment by JeremyRand** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

JeremyRand:
> Starting December 26, the `dev` branch has been failing on Travis
> CI.  See the Travis build history:
> https://travis-ci.org/namecoin/namecoin-core/builds

Pinging @domob1812 .

-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJYib1oAAoJELPy0WV4bWVwc8kQAL5EhLDPxkq04ksvlR/14OZT
A3FD09akDi+AfG4cHfvrIhIBvcD36IqruwxqNbIHnUkyjDRkloi+FVH0PtrnFZGt
l3zp2l948tzUFAiMw9VjXorwDmsUVh/aNEL73pjW5KQE8y2BcV33oqiJwWYa3de4
ez7WNHMpmMhCsGl22FvqX9UlLyTs3lfNjMwxaM8HL0h812FqhmHkm2HhFeUVTYMz
Vi7VEaCZs49xDNFvpATIdvt5Plh7MIk5Y7ZHvYlsouwQc0RgSOMl5p/sXtSNxSQn
+bhDV6MpRrK9/LDQ996TFpxgQgTL38ZYHh/m8km2B+EvNBshTYftPbE8lY8d7+ud
ZiX9PkP1vH5SlixTAQxs1HoVwJFSSGOMEV0cXo5yA2OiZgjNS0c4sc5SZ7vzWzfs
v+D4N2qB33XqBh0FUGHtZIKsvhoYd/1MvS90gOyw/JQQtGFC8nDI3w/pImB6V1ao
mFmk3GAXl8zw989tSrQ0aousHCjzczg9g8HJAKShBe/5kF6swpZ0YtETH71du/RR
IiHDLm1n/MwchzvyGuUUdp6HaLhMRKOYDnMTYQG6sIv5fVNlnzh3meBOuihtUwl8
WQ2uGOd8Bvn35C7nkknLxEXS8beIGoG1TA5MVNN32HyBj8fjJz4y3kcwTDl1+Arw
kg/QFGSd+JgMJfEEcE+y
=SpAP
-----END PGP SIGNATURE-----

**Comment by domob1812** [comment]
Skimming through one of the logs, I don't really see what the error is.  Do you know?

I'm always running the unit and regression tests manually before committing anything (and they succeed in the log I looked at), so maybe this is because of an upstream change in the Travis config that needs to be updated or so.

**Comment by JeremyRand** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Daniel Kraft:
> Skimming through one of the logs, I don't really see what the error
> is.  Do you know?
> 
> I'm always running the unit and regression tests manually before
> committing anything (and they succeed in the log I looked at), so
> maybe this is because of an upstream change in the Travis config
> that needs to be updated or so.

Upstream Bitcoin Core added a line `make distdir VERSION=$HOST` to
Travis.  For some reason, Namecoin Core is failing to perform this
build command.  See
https://travis-ci.org/namecoin/namecoin-core/jobs/186823962#L1567 .

The error appears to be this:

```
make[1]: *** No rule to make target `test/data/txcreatesignv1.hex',
needed by `distdir'.  Stop.
```

@domob1812 Any idea why that particular error is showing up for
Namecoin Core but not for upstream Bitcoin Core?


-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJYkmhfAAoJELPy0WV4bWVwbGwQAM2TFrTutMV1qSW/7EmyoWjG
Q8MsGDIfVibOvBOyKUg49nPmGatHYY7mDUJg5IlUgKaHJMt+EeiGGPqgPdlJhMpg
Tb1j8dcIlaK8L6aDnSXRs/66XrhPGwDtTs1oXE/EZhLRKcJwvl6Rdvln9hp1SoWz
Uyo/eXjFV0huj6EFaOWMwkyqtRsdXWvqIDgt7b7TYoKc/LN32ny930NQz0fdvIne
sJDQeHUY+3aMwRybuJtY0RccVGRPWyba+JBudJ8JuwTgPz2AJTxz5QQI9r6sbewk
eqNUnvAE3UzNxgOYIT3lHwhd0Dw/mOFuw5zJ5kzu3nHETYabIMvcQqCTknMV3juR
SkXwDNhv/F/O0AkKmg3vpG+yHICHiPuS1Hg14vDHDzze6anW8mljySvSg7hACYZ8
6pvnp6wplnc/ZzI3jpbwTKTRaOOsACZ3C7Skd/v2XH+I2F/XB2qgnQU+w/Lc6OG4
2aR6J6xs2ZKpQV9T5+UgZzlFYGdDg6PfTn3/auYIWbWEEM5J9BsE8cKIzvsqPUD7
2yakDzd4FiAop7ZpkmvGCT4DrrZhA3sKxoSm9cWjbrHyuu19zJFXv3R9Nwu2hk52
wrY7LJr6HsYJ5LT4pJEtGS8f8cnEm++N/EVPehH6u9tn7fDWPunuc8sOp0mLRaZH
n+KT6n+2lfnPEa0VqWgY
=Afoa
-----END PGP SIGNATURE-----

**Comment by domob1812** [comment]
Some time ago Bitcoin introduced tests for `bitcoin-tx` that change the version number of the created transactions.  Since a change here is incompatible to Namecoin (because Namecoin uses a fixed `nVersion` to distinguish Namecoin transactions; something I proposed to remove with a fork some time ago in the forum), I've "reverted" these changes.  Thus this file does not exist for Namecoin, it is instead `test/data/txcreatesign.hex`.

I don't know where exactly to fix this, but I guess at some point in some Makefile or so there's probably a list of files that needs to be updated.

**Comment by domob1812** [comment]
I assume this was fixed in #155.  Feel free to reopen if not.
